javalib/src/main/scala/java/net/URI.scala

```diff
@@ -1,33 +1,1415 @@
 package java.net
 
-final class URI(scheme: String,
-                userInfo: String,
-                host: String,
-                port: Int,
-                path: String,
-                query: String,
-                fragment: String)
-    extends Serializable {
+import java.io.IOException
+import java.io.Serializable
+import java.io.UnsupportedEncodingException
+import java.util.StringTokenizer
 
-  def getFragment(): String =
-    fragment
+object URI {
+  val unreserved: String = "_-!.~\'()*" //$NON-NLS-1$
```

Comment:
These values don't appear in the Javadoc, so I think they should be made `private`